extensions/common/azure/azure-blob-core/src/main/java/org/eclipse/edc/azure/blob/BlobStoreCoreExtensionConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2024 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.azure.blob;
+
+import org.eclipse.edc.runtime.metamodel.annotation.Setting;
+import org.eclipse.edc.runtime.metamodel.annotation.Settings;
+
+@Settings
+public record BlobStoreCoreExtensionConfig(
+        @Setting(key = "edc.azure.block.size.mb", description = "The block size, in mb, to parallel blob upload.", defaultValue = "4", required = false) long blockSize,
```

Comment:
when `defaultValue` is set, there's no need to set the `required` one (if there's a default value in fact the setting is never required)

Reply:
You're right, wanted to be more explicit, but is redundant. Removed in [here](https://github.com/eclipse-edc/Technology-Azure/pull/298/commits/a20a5bd83a4033c9249b24ca810cbacdc8a248df).